app/build.gradle

```diff
@@ -106,6 +110,11 @@ dependencies {
     compile "com.squareup.retrofit2:converter-gson:$retrofitVersion"
     compile "com.google.code.gson:gson:2.7"
 
+    // RX
+    compile "io.reactivex.rxjava2:rxjava:2.0.4"
```

Comment:
In this case none of these versions are repeated (rxjava has its own releases; same for rxandroid and rxjava adapter for retrofit). I will create variables anyway if it's easier to read, since the RX version is important, but in general I don't think we need to create variables for version numbers that don't repeat.